src/containers/shared/components/TransactionTable/styles.scss

```diff
@@ -0,0 +1,277 @@
+@import '../../../shared/css/variables';
+
+.account-transactions {
```

Comment:
This class should not be named this.  It should be a more general name like .transaction-table especially since these styles are used on the nft.

Reply:
true, will change